macros/shared

```diff
@@ -222,6 +222,13 @@ CROSS_COMPILATION_CONF_EOF\
 # failures when rewriting "shebangs" in crate files.
 %__brp_mangle_shebangs /bin/true
 
+# Do not compress debuginfo.
+%_find_debuginfo_dwz_opts %{nil}
+
+# Compress RPM payloads with zstd level 19 using 9 threads.
+%_source_payload w19T9.zstdio
+%_binary_payload w19T9.zstdio
```

Comment:
Can/should we use T0 to match the number of CPU cores instead?  The zstd CLI supports that, but I'm not sure if RPM passes it through directly; it seems to?

Out of curiosity, did you try different compression levels?  19 is near the max, and (naively) it seems like .rpm transfer out wouldn't be as slow as the CPU hit from near-max compression.

Reply:
The "threads" value is set via `ZSTD_CCtx_setParameter(_stream, ZSTD_c_nbWorkers, threads)`. The zstd manual says that 0 is the default value and means single-threaded mode.

I experimented with compression levels from 0 to 19 in steps of 5 or so. "0" was the fastest, and all the others were approximately one second slower.

I've now experimented with threads and found that single-threaded mode seems to be the fastest.